libgadget/blackhole.c

```diff
@@ -602,37 +930,68 @@ blackhole_accretion_ngbiter(TreeWalkQueryBHAccretion * I,
 
     /* Accretion / merger doesn't do self interaction */
     if(P[other].ID == I->ID) return;
-
-    if(P[other].Type == 5 && r2 < iter->accretion_kernel.HH)	/* we have a black hole merger */
+    
+    /* we have a black hole merger, use 2 times GravitationalSoftening as merging criteria*/
```

Comment:
Add a comment here explaining that the previous implementation uses the softening length.